src/site/content/en/blog/text-fragments/index.md

```diff
@@ -0,0 +1,414 @@
+No other of the three occurrences of the word `text` has the same surrounding words.
+Armed with this knowledge, I can tweak the previous URL and add the `prefix-` and the `-suffix`.
+Like the other parameters, they, too, need to be percent-encoded.
+<a href="https://blog.chromium.org/2019/12/chrome-80-content-indexing-es-modules.html#:~:text=the-,text,-parameter"><code>https://blog.chromium.org/2019/12/chrome-80-content-indexing-es-modules.html<strong>#:~:text=the-,text,-parameter</strong></code></a>.
+To allow the parser to clearly identify the `prefix-` and the `-suffix`,
+they need to be separated from the `textStart` and the optional `textEnd` with a dash&nbsp;`-`.
+
+<figure class="w-figure">
+  <img src="correct-text.png" alt="" class="w-screenshot">
+  <figcaption class="w-figcaption">Text Fragment matching at the correct occurrence of <code>Text</code>.</figcaption>
+</figure>
+
+### The full syntax
+
+The full syntax of Text Fragments is shown below (square brackets indicate an optional parameter).
+The values for all parameters need to be percent-encoded.
+This is especially important for the dash `-`, ampersand `&`, and comma `,` characters,
+so they are not being interpreted as part of the text directive syntax.
+
+```bash
+#:~:text=[prefix-,]textStart[,textEnd][,-suffix]
+```
+
+Each of `prefix-`, `textStart`, `textEnd`, and `-suffix` will only match text within a single
+[block-level element](https://developer.mozilla.org/en-US/docs/Web/HTML/Block-level_elements#Elements).
+For example, `:~:text=The quick,lazy dog` will fail to match in the following example,
+because the starting string "The quick" does not appear within a single,
+uninterrupted block-level element:
+
+```html
+  <div>The<div> </div>quick brown fox</div>
+  <div>jumped over the lazy dog</div>
+```
+
+It does, however, match in this example:
+
+```html
+  <div>The quick brown fox</div>
+  <div>jumped over the lazy dog</div>
+```
+
+### Multiple text fragments in one URL
+
+Note that multiple text fragments can appear on one URL.
+The particular text fragments need to be separated by an ampersand character `&`.
+Here is an example link with three text fragments:
+<a href="https://blog.chromium.org/2019/12/chrome-80-content-indexing-es-modules.html#:~:text=Text%20URL%20Fragments&text=text,-parameter&text=:~:text=On%20islands,%20birds%20can%20contribute%20as%20much%20as%2060%25%20of%20a%20cat's%20diet"><code>https://blog.chromium.org/2019/12/chrome-80-content-indexing-es-modules.html<strong>#:~:text=Text%20URL%20Fragments&text=text,-parameter&text=:~:text=On%20islands,%20birds%20can%20contribute%20as%20much%20as%2060%25%20of%20a%20cat's%20diet<strong></code></a>.
+
+
+### Mixing element and text fragments
+
+Traditional element fragments can be combined with text fragments.
+It is perfectly fine to have both in the same URL, for example,
+to provide a meaningful fallback for browsers that do not support Text Fragments yet.
```

Comment:
This doesn't really work for browsers that don't support text fragments, they won't know to strip out the `:~:` part so the browser won't find any fragment.

The more useful case here is if text on the page changes so the text fragment no longer matches, a conforming browser will then fall back to the element id which might be the nearest section header, for example.

Reply:
Changed to what you suggested.